File: Training-Site/Baekjoon/2/2000/2292/2292.go
package main

import (
	"bufio"
	"os"
	"strconv"
)

func main() {
	var sqrt int = 1
	var count int = 1

	input := bufio.NewScanner(os.Stdin)
	if input.Scan() {
		for {
			i, _ := strconv.Atoi(input.Text())
			if sqrt >= i {
				break
			}

			sqrt += 6 * count
			count++
		}
	}

	print(count)
}

File: Training-Site/Baekjoon/2/2000/2292/2292.cpp
#include <iostream>

using namespace std;

int main() {
    long long input = 0;
    long long sqrt = 1;
    float count = 1;

    cin>>input;

    while (true) {
        if (sqrt >= input)
            break;

        sqrt += 6 * count++;
    }

    cout<<count<<endl;
    return 0;
}